src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -7,23 +7,39 @@
 using System.CommandLine;
 using System.CommandLine.Invocation;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Text;
+using System.Management;
 using Process = System.Diagnostics.Process;
+using System.IO;
 
 namespace Microsoft.Internal.Common.Commands
 {
     public class ProcessStatusCommandHandler
     {
-        public static Command ProcessStatusCommand(string description) =>
-            new Command(name: "ps", description)
+        public static Command ProcessStatusCommand(string description)
+        {
+            var cmd = new Command(name: "ps", description)
             {
-                Handler = CommandHandler.Create<IConsole>(PrintProcessStatus)
+                Handler = CommandHandler.Create<IConsole, bool>(PrintProcessStatus),
+                
+
             };
+            cmd.AddOption(CommandLineOption());
+            return cmd;
+        }
 
+        private static Option CommandLineOption() =>
```

Comment:
I'm not sure we really need an option to enable the command line. Why not always print it?

/cc: @noahfalk